File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    y = []
    x = []
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == '#':
                y.append(i)
                y.append(i + 1)
                x.append(j)
                x.append(j + 1)

    x.sort()
    y.sort()

    answer = []

    answer.append(y[0])
    answer.append(x[0])
    answer.append(y[-1])
    answer.append(x[-1])

    return answer

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    
    var x = [];
    var y = [];
    
    for(var i=0; i<wallpaper.length; i++) {
        for(var j=0; j<wallpaper[0].length; j++) {
            if (wallpaper[i][j] == "#") {
                x.push(i);
                y.push(j);
            }
        }
    }
    console.log(x,y)
    x.sort((a,b) => a-b);
    y.sort((a,b) => a-b);
    
    return [x[0], y[0], x[x.length-1]+1, y[y.length-1]+1];
}